packages/mask/src/social-network-adaptor/twitter.com/injection/NFT/NFTAvatarInTwitter.tsx

```diff
@@ -248,11 +250,18 @@ function NFTAvatarInTwitter(props: NFTAvatarInTwitterProps) {
             })
         }
 
-        linkParentDom.addEventListener('click', handler, true)
+        const clean = () => {
+            linkParentDom.removeEventListener('click', handler, true)
+        }
 
-        return () => {
-            linkParentDom.removeEventListener('click', handler)
+        if (!nftAvatar) {
+            clean()
+            return
         }
+
+        linkParentDom.addEventListener('click', handler, true)
+
+        return clean
```

Comment:
```
if (nftAvatar) {
    linkParentDom.addEventListener('click', handler, true)
}

return () => {
    linkParentDom.removeEventListener('click', handler, true)

}
```

Reply:
We need clean the listener when `nftavatar` be `undefined`.